vars/junit2otel.groovy

```diff
@@ -55,33 +54,29 @@ populated later on, using the https://github.com/mdelapenya/junit2otel library.
 
 */
 def call(Map args = [:]) {
-  def sendTraces = env.JUNIT_2_OTLP?.trim() ? true : false
-
-  if (sendTraces) {
-    if(!isUnix()){
-      error('junit: windows is not supported yet.')
-    }
+  if(!isUnix()){
+    error('junit: windows is not supported yet.')
+  }
 
-    if(!isInstalled(tool: 'docker', flag: '--version')) {
-      error('junit2otel: docker is not installed but required.')
-    }
+  if(!isInstalled(tool: 'docker', flag: '--version')) {
+    error('junit2otel: docker is not installed but required.')
+  }
 
-    withOtelEnv() {
-      def testResults = args.containsKey('testResults') ? args.testResults : error("junit2otel: testResults parameter is required.")
-      def serviceName = env.JENKINS_OTEL_SERVICE_NAME?.trim() ? env.JENKINS_OTEL_SERVICE_NAME?.trim()+"-junit" : (env.REPO?.trim() ? env.REPO?.trim() : 'junit2otel')
-      def serviceVersion = env.JUNIT_OTEL_SERVICE_VERSION?.trim() ? env.JUNIT_OTEL_SERVICE_VERSION?.trim() : calculateFallbackServiceVersion()
-      def traceName = env.JUNIT_OTEL_TRACE_NAME?.trim() ? env.JUNIT_OTEL_TRACE_NAME?.trim() : (env.REPO?.trim() ? env.REPO?.trim() : 'junit2otel')
+  withOtelEnv() {
+    def testResults = args.containsKey('testResults') ? args.testResults : error("junit2otel: testResults parameter is required.")
```

Comment:
By doing so, then https://github.com/elastic/apm-pipeline-library/pull/1629/files#diff-9de9abc4e2e6b84ce6d6cda7b44f73c8d7a66722272ef624c330ecbee01b9068R83 should be changed with the `args` but without `serviceName`...

Otherwise, the junit(args) will fail with the signature

Reply:
Done in 6e17ef63631d636892caf7bbcb08fc823559325d